Docs/Database/CREATE_DATABASE.sql

```diff
@@ -47,7 +47,7 @@ ENGINE = InnoDB;
 CREATE TABLE IF NOT EXISTS `mydb`.`ITEMS_MENU` (
   `ITEM_ID` INT NOT NULL,
   `MENU_ID` INT NOT NULL,
-  `AVAILABLE` ENUM("Y", "N") NOT NULL,
+  `AVAILABLE_MENU` ENUM("Y", "N") NOT NULL,
```

Comment:
Making changes to the database this late in the development means we have to hunt down every query call to this column and update its name. 
If it's conflicting with `Cart.Available`, you can specify which column you're calling in the SQL statement like `Items_Menu.Available`. Better to fix one place than N places.

Reply:
Agree, I'm rusty on SQL. That's an easier and simpler solution. I'll revert the change and update the request.